worker/echo/dbus/meson.build

```diff
@@ -0,0 +1,15 @@
+configure_file(
+  configuration: config_data,
+  input: 'com.redhat.Yggdrasil1.Worker1.echo.service.in',
+  output: '@BASENAME@',
+  install: true,
+  install_dir: dbus.get_variable(pkgconfig: 'system_bus_services_dir')
+)
```

Comment:
If the service file from this directory was deleted, then the configuration file should also not be installed.